rebasehelper/tests/test_specfile.py

```diff
@@ -448,3 +452,8 @@ def test_typo_fix_spec_hook(self):
         assert '- This is chnagelog entry with some indentional typos\n' in self.SPEC_FILE_OBJECT.spec_content
         spec_hooks_runner.run_spec_hooks(None, self.SPEC_FILE_OBJECT)
         assert '- This is changelog entry with some intentional typos\n' in self.SPEC_FILE_OBJECT.spec_content
+
+    def test_pypi_to_python_hosted_url_trans(self):
+        assert PypiUrlFixHook.get_name() in spec_hooks_runner.spec_hooks
+        spec_hooks_runner.run_spec_hooks(None, self.SPEC_FILE_OBJECT)
+        assert self.SOURCE_7 == self.SPEC_FILE_OBJECT.sources[6]
```

Comment:
I think you should test if the original URL is present before the hook is run and that it is fixed after.
You should also probably use `self.SPEC_FILE_OBJECT._get_raw_source_string(7)` instead of `self.SPEC_FILE_OBJECT.sources[6]`.